package exceptions;

public class _7 {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			int a=100/0;
		} finally {
			// TODO: handle finally clause
			System.out.println("error has hapened");
		}

	}

}
